huang-yuwei/pixi-cjk
# Breakable chars for CJK

# Why
- the space breaking rules in the original PIXI isn't correct, and it gets worse when using CJK

# Summary
- convert the Kinsoku-Shorui to an abstracted feature for using it in word-wrap
- create a breakable-char feature for determining the breaking space correctly
- overwrite the word-wrap feature

# Testing
- check whether after this PR, it doesn't break the current behaviors 
- check whether Kinsoku-Shorui still work after this PR
- check whether breaking spaces look normal in CJK

# Screenshots
### BEFORE
![image](https://user-images.githubusercontent.com/44968790/108849477-85777c80-7625-11eb-9627-112c0ac8f223.png)

### AFTER
![image](https://user-images.githubusercontent.com/44968790/108849509-8c9e8a80-7625-11eb-967b-7d030639011b.png)
